lambda/lambda-returns-usr-info.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+from botocore.exceptions import ClientError
+import psycopg2
+from psycopg2 import OperationalError
+
+
+def getCredentials():
+    credential = {}
+    
+    secret_name = "mysecretname"
```

Comment:
please take this two variables from an environment variable (region and secret name).